VSDiagnostics/VSDiagnostics/VSDiagnostics.Test/Tests/Async/SyncMethodWithAsyncSuffixTests.cs

```diff
@@ -0,0 +1,637 @@
+﻿using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using RoslynTester.Helpers.CSharp;
+using VSDiagnostics.Diagnostics.Async.SyncMethodWithSyncSuffix;
+
+namespace VSDiagnostics.Test.Tests.Async
+{
+    [TestClass]
+    public class SyncMethodWithAsyncSuffixTests : CSharpCodeFixVerifier
+    {
+        protected override CodeFixProvider CodeFixProvider => new SyncMethodWithAsyncSuffixCodeFix();
+
+        protected override DiagnosticAnalyzer DiagnosticAnalyzer => new SyncMethodWithAsyncSuffixAnalyzer();
+
+        [TestMethod]
+        public void SyncMethodWithAsyncSuffix_WithoutAsyncKeywordAndNoSuffix_DoesNotDisplayWarning()
```

Comment:
We've changed our "convention" and no longer suffix our tests with `DoesNotDisplayWarning` or `DisplaysWarning` -- can you remove these? The intent of the test is always clear from the way we verify diagnostics and there are only two possible outcomes: a failure of a test always requires us to dive into it anyway.